### PR TITLE
[T3A2][W11-A2] Zhi Jiang

### PR DESCRIPTION
**Pull request by zavfel**
Ready for Review

### CODE REVIEW

File: src/seedu/addressbook/commands/CommandResult.java [hunk 1]
```diff
@@ -32,5 +32,13 @@ public CommandResult(String feedbackToUser, List<? extends ReadOnlyPerson> relev
     public Optional<List<? extends ReadOnlyPerson>> getRelevantPersons() {
         return Optional.ofNullable(relevantPersons);
     }
+    
+    /**
+     * get feedbackToUser member
+     * @return feedbacktoUser
+     */
```
Comment (cmkumar87) [inline src/seedu/addressbook/commands/CommandResult.java]: pls remove auto gen header comments if they are not useful. Usually accessors don't need comments as their names are informative enough.

File: src/seedu/addressbook/ui/TextUi.java [hunk 2]
```diff
@@ -126,7 +126,7 @@ public void showResultToUser(CommandResult result) {
         if(resultPersons.isPresent()) {
             showPersonListView(resultPersons.get());
         }
-        showToUser(result.feedbackToUser, DIVIDER);
+        showToUser(result.getFeedback(), DIVIDER);
     }
```
Comment (cmkumar87) [inline src/seedu/addressbook/ui/TextUi.java]: good catch

**Comment by cmkumar87** [comment]
@zavfel  Please acknowledge and close PR. We are now advised to follow this protocol for PR closing to ensure student read our comments.

**Comment by zavfel** [comment]
Noted, made the necessary changes.

**Pull request closed without merging**